packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -142,6 +188,30 @@ export default function ReusableBlockConvertButton( {
 										value={ title }
 										onChange={ setTitle }
 									/>
+									<SelectControl
+										label={ __( 'Category' ) }
+										onChange={ setCategoryId }
+										options={ categoryOptions }
+										size="__unstable-large"
+										value={ categoryId }
+									/>
+
+									<ToggleControl
+										label="Synced"
```

Comment:
```suggestion
										label={ __( 'Synced' ) }
```

I missed this in my earlier review, sorry.

Reply:
Thanks, fixed.